common/changes/@microsoft/rush/sachinjoseph-pnpm1890-workaround_2019-06-25-06-48.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "Workaround for pnpm 1890",
```

Comment:
Please provide a more clearly worded changelog. See https://rushjs.io/pages/best_practices/change_logs/

Reply:
Will do

Reply:
Done

Reply:
If you ran `rush change` a second time (rather than editing the json file) don't forget to delete the old change file

Reply:
oh, okay. let me do that.

Reply:
Done